### DISCUSSION TITLE
Where's the settings tab-?

### DISCUSSION THREAD

**Discussion by TimWasNotHere** [fast4x/RiMusic#1530] [Q&A]
Since the last update, I legitimately cannot find the settings tab.
![Screenshot_20240415-065730.png](https://github.com/fast4x/RiMusic/assets/98356450/aaf7198f-b4b0-4948-b8af-6aa490dfda4d)

I see the floating search icon with a settings icon next to it, but it only brings me to search. Sorry for the stupid question, I dunno if this is a bug or am I just missing something obvious.

**Comment by OrangeZXZ** [comment]
> Since the last update, I legitimately cannot find the settings tab.
> ![Screenshot_20240415-065730.png](https://github.com/fast4x/RiMusic/assets/98356450/aaf7198f-b4b0-4948-b8af-6aa490dfda4d)
> 
> I see the floating search icon with a settings icon next to it, but it only brings me to search. Sorry for the stupid question, I dunno if this is a bug or am I just missing something obvious.

You need to double tap the button Next update this feature will be removed
